internal/mode/static/state/conditions/conditions.go

```diff
@@ -57,6 +57,12 @@ const (
 	RouteMessageFailedNginxReload = GatewayMessageFailedNginxReload + ". NGINX may still be configured " +
 		"for this HTTPRoute. However, future updates to this resource will not be configured until the Gateway " +
 		"is programmed again"
+	// RouteConditionPartiallyInvalid is a condition which indicates that the Route contains
+	// a combination of both valid and invalid rules.
+	//
+	// FIXME(Jee): Update to Gateway sig v1 version when released.
```

Comment:
for consistency, let's use github ids
```suggestion
	// FIXME(bjee19): Update to Gateway sig v1 version when released.
```